pipdeptree.py

```diff
@@ -267,10 +306,12 @@ class PackageDAG(Mapping):
 
     @classmethod
     def from_pkgs(cls, pkgs):
-        pkgs = [DistPackage(p) for p in pkgs]
+        pkgs = [DistPackage(p) for p in pkgs if p.key != 'pkg-resources']
```

Comment:
Why is 'pkg-resources' omitted? In case it's required, it'd be good to have a comment explaining why.  Just let me know the reason in that case, I'll add add the comment after merging the PR

Reply:
I guess it could be left in. I forgot that I added that. The only reason I did it was that it's not even on pypi.org, I think it's being installed as part of pip. It's always alone and without dependencies and shows a version of 0.0.0 which is weird.
But I don't mind deleting that if clause if you deem it a better option.